contracts/swaps/connectors/SwapsImplSovrynSwap.sol

```diff
@@ -227,8 +223,10 @@ contract SwapsImplSovrynSwap is State, ISwapsImpl {
     ) public view returns (uint256) {
         ISovrynSwapNetwork sovrynSwapNetwork =
             getSovrynSwapNetworkContract(sovrynSwapContractRegistryAddress);
```

Comment:
it shadows existing variable 

all params can be added leading underscores

Reply:
Changed the variable name for better unification